packages/eslint-plugin-wpcalypso/package.json

```diff
@@ -28,6 +28,7 @@
 		"eslint-plugin-jsdoc": "^30.7.3"
 	},
 	"devDependencies": {
+		"eslint": ">=7.12.0",
```

Comment:
Just curious, how did you decide on `>=` rather than `^` which I usually see?

Reply:
I copied the existing `peerDependency`